Tests/RegexTests/LegacyTests.swift

```diff
@@ -133,12 +133,8 @@ private func performTest<Capture>(
   let legacyProgram = try! compile(ast)
   run(TortoiseVM(program: legacyProgram), name: "Lonesome George")
   run(HareVM(program: legacyProgram), name: "Harvey")
-  // TODO: Support captures in the matching engine.
-  guard !ast.hasCapture else {
-    return
-  }
-  let program = try! Compiler(ast: ast).emit()
-  run(Executor(program: program), name: "Matching Engine")
+//  let program = try! Compiler(ast: ast).emit()
+//  run(Executor(program: program), name: "Matching Engine")
```

Comment:
Should this be enabled?

Reply:
I should probably try to rebase this cleanly on main instead of your work. If we can formally layer your approach on top that might be cleaner.